airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/typing_deduping/Sql.kt

```diff
@@ -122,12 +122,10 @@ data class Sql(val transactions: List<List<String>>) {
                     .map { transaction: List<String> ->
                         transaction
                             .stream()
-                            .filter { statement: String? ->
-                                statement != null && !statement.isEmpty()
-                            }
-                            .map { statement: String ->
+                            .filter { statement: String? -> !statement.isNullOrEmpty() }
+                            .map internalMap@{ statement: String ->
```

Comment:
what is internalMap?

Reply:
https://kotlinlang.org/docs/returns.html#return-to-labels